VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common version |
 |---------|-------------|-----------------|----------------|
+| 3.2.0-amazon.alpha.5  | 5.0.0-amazon.alpha.5.billing3.iap2 | 3.14.0 | 2.0.0-amazon.alpha.5.billing3.iap2 (Android) /2.0.0 (iOS) |
```

Comment:
i think android and ios version are flipped here

Reply:
```suggestion
| 3.2.0-amazon.alpha.5  | 3.14.0 | 5.0.0-amazon.alpha.5.billing3.iap2 | 2.0.0-amazon.alpha.5.billing3.iap2 (Android) /2.0.0 (iOS) |
```